src/renderer/components/dock/logs.tsx

```diff
@@ -51,112 +68,176 @@ export class Logs extends React.Component<Props> {
   }
 
   componentDidMount() {
-    disposeOnUnmount(this,
-      reaction(() => this.props.tab.id, this.reload, { fireImmediately: true }),
-    );
+    disposeOnUnmount(this, [
+      kubeWatchApi.subscribeStores([
+        podsStore,
+      ], {
+        namespaces: logTabStore.getNamespaces(),
+      }),
+      when(() => this.canLoad, () => this.load(true)),
+      disposingReaction(
```

Comment:
I would move all of this reactions to different methods for sake of readability. Now it's too big for `componentDidMount()`?

Reply:
sounds good